pkg/util/cluster/cluster.go

```diff
@@ -520,7 +520,7 @@ func (c *Cluster) registerSubscription(ctx context.Context) error {
 			TenantID: c.env.TenantID(),
 			RegisteredFeatures: []api.RegisteredFeatureProfile{
 				{
-					Name:  "Microsoft.RedHatOpenShift/RedHatEngineering",
+					Name:  "Microsoft.RedHatOpenShift/BYONsg",
```

Comment:
why is this being changed?

Reply:
It shouldn't be there... got spilled from my testing. sorry about that!

Reply:
Removed now.